lib/openstax/aws/stack.rb

```diff
@@ -1,11 +1,12 @@
 module OpenStax::Aws
   class Stack
 
-    attr_reader :name, :id, :absolute_template_path, :dry_run,
+    attr_reader :name, :tags, :id, :absolute_template_path, :dry_run,
                 :enable_termination_protection, :region, :parameter_defaults,
                 :volatile_parameters_block, :secrets_blocks
 
-    def initialize(id: nil, name:, region:, enable_termination_protection: false,
+    def initialize(id: nil, name:, tags: {},
```

Comment:
here, tags is defaulted using a regular hash, not an indifferent hash?

Reply:
It is ok here because this `tags` is not modified, and an empty hash and `hash[:foo]` and `hash["foo"]` will both be nil (correctly)